backend/cognito/cognito.template.yml

```diff
@@ -32,8 +38,37 @@ Globals:
     Timeout: 10
     Runtime: nodejs18.x
     PermissionsBoundary: !If [ UsePermissionsBoundary, !Ref PermissionsBoundary, !Ref AWS::NoValue ]
+    Environment:
+      Variables:
+        NOTIFY_API_KEY: !Ref NotifyApiKey
```

Comment:
Use a dynamic reference here to the Secrets Manager

Reply:
addressed